src/pseudopeople/utilities.py

```diff
@@ -72,7 +72,10 @@ def get_index_to_noise(
     """
 
     # Get rows to noise
-    not_empty_idx = column.index[(column != "") & (column.notna())]
+    if isinstance(data, pd.Series):
+        not_empty_idx = data.index[(data != "") & (data.notna())]
+    else:
+        not_empty_idx = data.index
```

Comment:
Why is this necessary? To handle row-level noise?

Reply:
Yes.  Allows for the same operation on a dataframe.